airbyte-integrations/connectors/source-zendesk-talk/integration_tests/spec.json

```diff
@@ -0,0 +1,49 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/zendesk-talk",
+  "changelogUrl": "https://docs.airbyte.io/integrations/sources/zendesk-talk",
+  "connectionSpecification": {
+    "title": "Zendesk Talk Spec",
+    "type": "object",
+    "properties": {
+      "subdomain": {
+        "title": "Subdomain",
+        "description": "The subdomain for your Zendesk Talk.",
+        "type": "string"
+      },
+      "access_token": {
+        "title": "Access Token",
+        "description": "The value of the API token generated. See the <a href=\"https://docs.airbyte.io/integrations/sources/zendesk-talk\">docs</a> for more information.",
+        "airbyte_secret": true,
+        "type": "string"
+      },
+      "email": {
+        "title": "Email",
+        "description": "The user email for your Zendesk account.",
+        "type": "string"
+      },
+      "start_date": {
+        "title": "Start Date",
+        "description": "The date from which you'd like to replicate data for Zendesk Talk API, in the format YYYY-MM-DDT00:00:00Z.",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
+        "examples": [
+          "2017-01-25T00:00:00Z"
+        ],
+        "type": "string",
+        "format": "date-time"
+      }
+    },
+    "required": [
+      "subdomain",
+      "access_token",
+      "email",
+      "start_date"
+    ]
+  },
+  "supportsIncremental": true,
+  "supportsNormalization": false,
+  "supportsDBT": false,
+  "supported_destination_sync_modes": [
+    "append"
+  ],
+  "authSpecification": null
```

Comment:
we can probably omit this for now if it is null

Reply:
this is how it is stored in `ConnectorSpecification`, don't think this has any effect on behavior since this file is just for SAT